### ISSUE TITLE
Tests not working

### ISSUE THREAD

**Issue by franroa** [GustavEikaas/easy-dotnet.nvim#30]
Hi there! very cool plugin!
I have realized that the tests runs are not working on my setup and is due to the hardcoded `./src` in these lines:

https://github.com/GustavEikaas/easy-dotnet.nvim/blob/main/lua/easy-dotnet/test-runner/keymaps.lua#L172
and
https://github.com/GustavEikaas/easy-dotnet.nvim/blob/main/lua/easy-dotnet/test-runner/keymaps.lua#L45

Maybe a vim variable setting the root of the tests?

**Comment by GustavEikaas** [comment]
Hello! 
First of all, thanks for trying this plugin.

I looked at the issue and made a pull request for fixing it. I coukd have it as a vim variable but I suspect most users will have multiple projects and it might be unnecessary hassle for the user to update this variable. 

Therefore I tried automatically finding your solution file or csproj file. Are you using lazy package manager, in that case can you test the pull request to see if it works for you?
Or maybe your package manager supports specifying commit ids anyways?
```lua
return {
  "GustavEikaas/easy-dotnet.nvim",
  commit = "f3cf1e2a0b5e187d3e1478fb062d99a8b5cfcf42",
  -- dir = "C:\\Users\\Gustav\\repo\\easy-dotnet.nvim",
  dependencies = { "nvim-lua/plenary.nvim", 'nvim-telescope/telescope.nvim', },
  config = ....
```

**Comment by franroa** [comment]
Hi this awesome, you are pretty fast!
Yes, it is working now for me with that commit
Thank you so much!

**Comment by GustavEikaas** [comment]
Amazing, merged it now. So you can remove the commit and use the latest from main.


Fixed by #31
